presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -134,6 +139,22 @@ private void insertData(byte[] data)
         }
     }
 
+    private void createMessagesWithHeader(String topicName)
+    {
+        try (KafkaProducer<byte[], byte[]> producer = createProducer()) {
+            ProducerRecord<byte[], byte[]> record = new ProducerRecord<byte[], byte[]>(topicName, null, "{}".getBytes(Charset.forName("UTF-8")));
+            record.headers()
+                    .add("notfoo", null);
```

Comment:
Can you change the value to something other than `null`?  As the test is written, it's not clear whether the `null` in the result comes from this header or the one a few lines below (`.add("foo", null)`).

Reply:
good point.